go/cmd/ccql/main.go

```diff
@@ -35,6 +37,7 @@ func main() {
 	help := flag.Bool("help", false, "Display usage")
 	user := flag.String("u", osUser, "MySQL username")
 	password := flag.String("p", "", "MySQL password")
+	askPassword := flag.Bool("P", false, "Ask mySQL password")
```

Comment:
Can we please change argument to `ask-pass`? This will make it compatible with `gh-ost` and with Percona Toolkit.

Reply:
Sure. Will change arg